modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -1572,7 +1590,7 @@ sub pipeline_analyses {
               '(1, "annotation.provider_name", "'.$self->o('annotation_provider_name').'"),'.
               '(1, "annotation.provider_url", "'.$self->o('annotation_provider_url').'"),'.
               '(1, "species.production_name", "'.$self->o('production_name').'"),'.
-              '(1, "repeat.analysis", "'.$self->o('full_repbase_logic_name').'"),'.
+	      (!($self->o('replace_repbase_with_red_to_mask')) ? '(1, "repeat.analysis", "'.$self->o('full_repbase_logic_name').'"),': '').
```

Comment:
RED will still need to have this meta key inserted

Reply:
Just added it.